### ISSUE TITLE
Library naming issue

### ISSUE THREAD

**Issue by ErcinDedeoglu** [autoNumeric/vue-autoNumeric#29]
https://github.com/autoNumeric/vue-autoNumeric/blob/252d616d2f8c0817674e785c89e07f4a0c501728/dist/vue-autonumeric.js#L8

https://github.com/autoNumeric/vue-autoNumeric/blob/252d616d2f8c0817674e785c89e07f4a0c501728/dist/vue-autonumeric.js#L12

require("AutoNumeric") looking for wrong way. it should be renamed to require("autonumeric") (lowercase)
When I change it to lowercase it start working.

Any idea?

**Comment by truesteps** [comment]
It's a bug, please fix it :D

**Comment by AlexandreBonneau** [comment]
The files in `dist` are auto-generated by Webpack.
On my end, `vue-autonumeric` works like a charm with that `require`.
I'll have to check if that require change does no break existing softwares.

**Comment by truesteps** [comment]
Yes, for me on MAC it works as a charm as well but as soon as I try to deploy it on any Linux machine (which is formatted to be Case sensitive), the build fails. The require in the component is fine but when webpack builds it, either browserify or babel imports its as `AutoNumeric` rather than `autonumeric` and case sensitive filesystems will always fail to locate it.
